CHANGELOG.md

```diff
@@ -2,12 +2,20 @@
 
 ## [Unreleased]
 
+### Added
+
+- Added a new API call, `PUT /metrics`, for configuring the metrics
+  system.
+- Added `app_name` field in InstanceInfo struct for storing the
```

Comment:
Yes. It is a breaking change for the API as well because now a get on `/` will also return the `app_name`. Which means that we should also update the swagger file to add the `app_name` in the `InstanceInfo` description.

Reply:
It is already updated. I was looking at the first commit **face palm**. In any case, we should keep this entry in the changelog

Reply:
Ok. Understood.